src/sds_data_model/dataframe.py

```diff
@@ -0,0 +1,254 @@
+from logging import getLogger, INFO, StreamHandler, Formatter
+from typing import Any, Dict, Optional,  Type, TypeVar
+from typing_extensions import Self
+from pathlib import Path
+from inspect import ismethod, signature
+
+from dataclasses import dataclass
+from typing_extensions import Self
+from pyspark.pandas import read_excel
+from pyspark.sql import DataFrame as SparkDataFrame
+
+from pyspark.pandas import DataFrame, read_excel
+from pyspark.sql import SparkSession
+from pyspark_vector_files import read_vector_files
+from pyspark_vector_files.gpkg import read_gpkg
+
+from pyspark.sql import SparkSession
+
+from sds_data_model._vector import _get_metadata, _get_name
+from sds_data_model.metadata import Metadata
+from sds_data_model._dataframe import _to_zarr_region, _create_dummy_dataset
+
+from pandas import DataFrame
```

Comment:
It's probably safest to alias this one too:

```suggestion
from pandas import DataFrame as PandasDataFrame
```